README.md

```diff
@@ -1,2 +1,228 @@
-# satellite6-upgrade
-Satellite 6 upgrade related tools
+# Satellite6-upgrade
+
+Satellite6 upgrade is a set of tools to be used for following reasons:
```

Comment:
s/to ensure data integrity post upgrade/to ensure data integrity after upgrade?

Not a big deal but just to avoid using post again.